conf/layer.conf

```diff
@@ -30,7 +30,7 @@ UPDATEHUB_IMAGE_TYPE_updatehub-imx ??= "active/inactive"
 UPDATEHUB_INSTALL_MODE_updatehub-imx ??= "tarball"
 
 PREFERRED_PROVIDER_virtual/bootloader_updatehub-imx ??= "u-boot-fslc"
-PREFERRED_RPROVIDER_u-boot-fw-utils_updatehub-imx ??= "u-boot-fslc-fw-utils"
+PREFERRED_RPROVIDER_u-boot-fw-utils_updatehub-imx ??= "libubootenv"
```

Comment:
I think here you need to add the `libubootenv-bin` otherwise we don't get the command-line utilities.